types/passport-qq/index.d.ts

```diff
@@ -0,0 +1,21 @@
+import { Profile } from "passport";
+import { Strategy as OAuth2Strategy } from "passport-oauth2";
+
+export interface Options {
+    clientID: string;
+    clientSecret: string;
+    callbackURL: string;
+    authorizationURL?: string;
+    tokenURL?: string;
+    scopeSeparator?: string;
+}
+
+export type Callback = (err: Error | null, user: unknown) => void;
```

Comment:
If desirable, precise types for `user` could likely be added based on this: <https://github.com/qdsang/passport-qq/blob/23697a318dfac0d8ea0dd12b6ca66ed3efa05255/lib/passport-qq/strategy.js#L104-L109>